diagram.md

```diff
@@ -170,7 +170,7 @@ The `Chapters Element` section lists all of the Chapters. Chapters are a way to
 ```
 Figure: Representation of the `Chapters Element` and a selection of its `Descendant Elements`.
 
-The `Cluster Elements` contain all of the video frames and audio for each track. In a given Matroska file, there are usually many `Cluster Elements`. The Clusters help to break up the `SimpleBlock` or `BlockGroup Elements` and help with seeking and error protection. It is RECOMMENDED the size of each individual `Cluster Element` be limited to store no more than 5 seconds or 5 megabytes. Every Cluster contains a timecode, usually the timecode that the first Block in the Cluster SHOULD be played back, but it doesn't have to be. Then there are one or more (usually many more) `BlockGroups` or `SimpleBlocks` in each Cluster. A BlockGroup can contain a Block of data, and any information relating directly to that Block.
+The `Cluster Elements` contain all of the video frames and audio for each track. In a given Matroska file, there are usually many `Cluster Elements`. A Matroska file SHOULD contain at least one `Cluster Elements`. The `Clusters` help to break up the `SimpleBlock` or `BlockGroup Elements` and help with seeking and error protection. It is RECOMMENDED that the size of each individual `Cluster` be limited to store no more than 5 seconds or 5 megabytes. Every `Cluster` MUST contains a `Timecode Element`. This SHOULD be the `Timecode` used to play the first `Block` in the `Cluster`. There SHOULD be one or more `BlockGroups` or `SimpleBlocks` in each `Cluster`. A `BlockGroup` MAY contain a `Block` of data and any information relating directly to that `Block`.
```

Comment:
Hmm, and I thought the Oxford comma had recently fallen out of recommended use but this Wikipedia article indicates that it is still favorable use, here in the States for American English? But internationally, not so much? I'm gonna... take it out.